js/chmod_test.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import { test, testPerm, assert, assertEqual } from "./test_util.ts";
+import * as deno from "deno";
+testPerm({ write: true }, function chmodSyncSuccess() {
+  const enc = new TextEncoder();
+  const data = enc.encode("Hello");
+  const filename = deno.makeTempDirSync() + "/test.txt";
+  deno.writeFileSync(filename, data, 0o666);
+  let fileInfo = deno.statSync(filename);
+  console.log(fileInfo.mode, 0o666); // This is printing 33206 438
```

Comment:
@ry deno.statSync(filename).mode is not working properly.

Checkout the line 802 in the build log [Here](https://travis-ci.com/denoland/deno/jobs/147768131). 
802: 33206 438. This is coming from the line console.log(fileInfo.mode, 0o666); on which I am commenting. 
I had initially opened an issue https://github.com/denoland/deno/issues/756. At that time it worked on travis. Now it is not working on travis also. Can you check if you are also having the same behaviour.